asset/scss/style.scss

```diff
@@ -0,0 +1,466 @@
+	background-color: $white;
+	.customer-head{
+		font-size: 48px;
+		color: #444444;
+		font-family: Roboto-light;
+		margin-top: 100px;
+		.ti-face-smile{
+			font-size: 36px;
+			margin-right: 19px;
+		}
+	}
+	.content-1{
+		font-size: 14px;
+		font-style: italic;
+		line-height: 19px;
+		margin-top: 15px;
+	}
+	.img{
+		margin-top: 100px;
+		text-align:center;
+		img{
+			max-width: 100%;
+		}
+	}
+	.customer-content{
+		line-height: 24px;
+		color: $black;
+		text-align: justify;
+		font-size: 13px;
+		.customer-detail-1{
+			margin-top: 26px;
+			.ti-unlock{
+				font-size: 18px;
+				color: $green;
+			}
+		}
+		.customer-detail-2{
+			margin-top: 17px;	
+			.ti-cup{
+				font-size: 18px;
+				color: $green;
+			}
+			.ti-stats-up{
+				font-size: 18px;
+				color: $green;
+			}
+		}
+	}
+	.button-customer{
+		margin-bottom: 100px;
+  	margin-top: 30px;
+	}
+}
+
+.rectangle4 {
+  height: 544px;
+  background-image: url(../images/Rectangle_3.jpg);
+  background-color: rgba(40,50,56,0.9);
+  .head-rec{
+  color: $white; 
+  font-family: Roboto-light;  
+  font-size: 48px;  
+  height: 63px;
+  margin-top: 50px;
+  text-align: center;
+	}
+	.owl-carousel{
+		.owl-stage-outer {
+  		height: 155px;
+		}
+		.owl-item{
+  		img{
+  			width: 53% !important;
+  			padding-top: 10px;
+  			margin: auto;
+			}
+		}
+	}
+	.our-customers {
+  	height: 127px;
+  	margin-top: 39px;
+  	font-family: Roboto-regular; 
+  	.cus-name{
+  		color: $white; 
+  		font-size: 13px;  
+  		font-weight: 300; 
+  		line-height: 14px;
+  		text-align: center;
+  		margin-top: 16px;
+  		display: block;
+		}
+		.cus-job{
+  		display: block;
+  		color: $white;
+  		font-weight: 300;
+  		line-height: 14px;
+  		font-size: 10px;
+  		text-align: center;
+  		padding-top: 2px;
+			}
+		.avata {
+ 			@extend %transition;
+ 			&:hover{
+ 				@include scale(1.2);
+ 			}
+ 			&:hover + .cus-name{
+ 				margin-top: 25px;
+ 			}
+		}
+	}
+	.ti-quote-left{
+		font-size: 72px;
+		color: $white;
+		margin-top: 39px;
+	}
+	.tes-text{
+		color: $white; 
+  	font-family: Roboto-light; 
+  	font-size: 18px;
+  	margin-top: 104px;
+  	line-height: 30px;
+  	text-align: justify;
+	}
+  .ti-quote-right{
+		font-size: 72px;
+		color: $white;
+		margin-top: -25px;
+	}
+}
+
+footer{
+	margin-left: -2px;
+	border: 1px solid #ECEFF1;
+	font-family: Roboto-regular; 
+	background-color: #39474E;
+	box-shadow: 0 15px 35px 0 rgba(0,0,0,0.1);
+	.row{
+		margin-top: 45px;
+		.col-1{
+			margin-top: -7px;
+			img{
+				width: 96px;
+			}
+		}
+		.col-2{
+			p{
+				color: $green;
+				font-size: 13px;
+				line-height: 18px;
+			}
+			a{
+				font-size: 10px;
+				color: #D0D8DC;
+			}
+		}
+		.col-3{
+			margin-top: 43px;
+			ul{
+				li{
+					float: left;
+					a{
+						color: $green; 
+						font-size: 13px; 
+						padding-left: 0px;
+					}
+					.ti-twitter-alt,.ti-youtube,.ti-linkedin{
+							margin-left: 37px;
+					}
+				}
+			}
+		}
+		.copyright{
```

Comment:
copy right sao phai set mac dinh the nay ak

Reply:
a chia n ra thành row riêng nên set riêng